### ISSUE TITLE
Auction start

### ISSUE THREAD

**Issue by slvstr1** [ECON3/dAuction2_interactive3#31]
When the auction starts, it takes a while to upload everything. Quite a lot of time has already passed once the auction starts. This is OK, but the timer should start once all have fully loaded screens. I know there is some jquery command for this (onload.window?). Could you perhaps have a jquery that gives a signal once the full window has been loaded? I will then make the timer start exactly at that moment.

**Comment by janvavra** [comment]
So you mean, that clients will load, then they will sent some post request to server to change some value in db and when this value is changed for all players, timer will start?

**Comment by slvstr1** [comment]
Yes, I think that is the way to do it. I can create a boolean called auction.all_ready and they will start once this boolean is True

**Comment by slvstr1** [comment]
I call it auction.auction_started and it has been implemented.

**Comment by janvavra** [comment]
Yes.
Idea is, that every player has player.ready set initially to false. When his interface loads, there will be POST request to DB and set it to true.
In addition, there will be a process running in ajax_admin, that will check, if all player.ready are true and then set auction_started to true and auction starts.
Or experimenter can view this readiness in client table, and then fire it manually.

**Comment by slvstr1** [comment]
Player.player_ready is implemented. Please implement: "When his interface loads, there will be POST request to DB and set it to true."

**Comment by slvstr1** [comment]
I added an auction ready action and state and then an auction start. Auction ready starts up the auction but doesnt start the timer (and some other internal stuff)

Please implement: "When his interface loads, there will be POST request to DB and set Player.player_ready to true."
Perhaps show the screen you also use for waiting saying "wait untill the auction starts"

**Comment by janvavra** [comment]
Please implement: "When his interface loads, there will be POST request to DB and set Player.player_ready to true." done

"Perhaps show the screen you also use for waiting saying "wait untill the auction starts"
I can do that, but what changes in db when auction in started?

**Comment by slvstr1** [comment]
When the auction screen is opening for all players, but the auction hasnt started yet (and the timer hasnt started yet), at this point the following conditions hold:
auction.auction_ready=True
auction.auction_started=False

When the auctions starts and the clock ticking
auction.auction_ready=True
auction.auction_started=True
